bucky.coffee

```diff
@@ -83,7 +83,7 @@ exportDef = ->
         else if tagOptions[key]?.toString().toLowerCase() is 'false'
           tagOptions[key] = null
    
-  options = extend {}, defaults, tagOptions
+  options = extend {}, tagOptions, defaults
```

Comment:
Either I'm crazy or `extend` is incorrectly implemented, b/c the original implementation absolutely should result in `tagOptions` overriding `defaults`.